org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
```

Comment:
is this an official GNOME project? Merely being hosted there doesn't mean it is.

Reply:
I would like to see org.gnomeradio.Radio or org.gnome.Radio being published on Flathub.

GNOME Circle required a Flathub page, so to become an official GNOME Project, the GNOME Foundation requires their matured apps to have a Flathub page.  See https://gitlab.gnome.org/Teams/Circle/-/issues/147

I have authored, written and published the source for GNU Network Object Model Environment Radio 64 (https://download.gnome.org/sources/gnome-radio/64/gnome-radio-64.0.tar.xz) and maintained it between 2017-2022.  Will the new maintainer step up?

It took 1 month of valgrind/gcc -S org.gnome.Radio.c retroinspection work to become fundamentally stable running with champlain, gst and gst-player in gtk_main_loop() IIRC and org.gnome.Radio streams BBC Radio by Greenwich time.  Perhaps some kind of clock would be feasible to implement on top of the new GTK code in gnome-radio/gtk/

You can view the latest code for org.gnome.Radio as C/XML/Makefile files on http://www.gnomeradio.org/~ole/radio/ and on https://gitlab.gnome.org/oka/gnome-radio.git with the GNU autoconf/automake/m4 and meson.build file for the gnome-radio binary.

Reply:
`org.gnomeradio.Radio` would definitely be accepted for the app-id, it meets the requirements.

Reply:
Ok, modified.  Do I need to resubmit the package?

Reply:
you need to rename the manifst to match. There is absolutely no reason to open a fifth PR.

Reply:
What should I rename the manifest to?

I have three files:

README.md
org.gnomeradio.Radio.appdata.xml
org.gnomeradio.Radio.json